src/Copy/BrandCopy.elm

```diff
@@ -447,88 +447,88 @@ brandCopy key =
-            CopyText "✨"
+            CopyText "Check back soon"
 
         StoryOneTitle ->
-            CopyText "✨"
+            CopyText "Diane"
 
         StoryOneTeaser ->
-            CopyText "✨"
+            CopyText "Diane felt her complaint about her husband’s treatment was badly handled, but a weight was lifted when she found an advocate."
 
         StoryOneTeaserImageAlt ->
             CopyText "✨"
 
         StoryOne1Quote ->
-            CopyText "✨"
+            CopyText "When my husband was first diagnosed with cancer, the treatment was amazing, he was in chemotherapy within 36hours. But later, an A&E visit went very badly when protocols and procedures were ignored. We put in a complaint to the Health Care Trust, and one sentence in their reply made us so angry."
 
         StoryOne1ImageAlt ->
             CopyText "✨"
 
         StoryOne1Message ->
-            CopyText ""
+            CopyText "They said my husband’s treatment had been appropriate but we knew the protocol for neutropenic sepsis."
 
         StoryOne2Quote ->
-            CopyText "✨"
+            CopyText "The leaflet about an advocate came with the reply. I had spoken to my husband. “What do you reckon, it’s a charity, we don’t have to but shall we give it a go?”. There were lots of forms but it was all very smooth. It was actually the week of my husband’s funeral when the worker called to arrange the first meeting, but I thought “I really do want to get on with this”."
 
         StoryOne2ImageAlt ->
             CopyText "✨"
 
         StoryOne2Message ->
-            CopyText "✨"
+            CopyText "You feel like you can’t say anything, it's the medical profession, they will close ranks against you."
 
         StoryOne3Quote ->
-            CopyText "✨"
+            CopyText "I had this real sense of relief that I was going to see somebody and when I met Susan, our advocate, it was so reassuring. She handed me a box of tissues and put together a letter in my name. Later, before the response meeting we met and she went through it with me, asking me which key things I wanted to remember to tell them."
 
         StoryOne3ImageAlt ->
             CopyText "✨"
 
         StoryOne3Message ->
-            CopyText ""
+            CopyText "You do worry that you might end up with a marker against your name."
 
         StoryOne4Quote ->
-            CopyText "✨"
+            CopyText "When we got to the response meeting it was amicable. They made big changes to the neutropenic sepsis process. Nurses can administer the intravenous drugs now, not just doctors and there are posters all over the hospital. I just felt like an enormous weight had been lifted - that’s all we’d wanted, them to admit that it wasn’t right."
 
         StoryOne4ImageAlt ->
             CopyText "✨"
 
         StoryOne4Message ->
-            CopyText "✨"
+            CopyText "I met someone who said “it’s thanks to you and your husband that I’m alive”."
```

Comment:
Check quote render.

Reply:
no issue on Chrome and Firefox for linux (fedora 30), latest versions. I guess if anything, it will be uncovered during testing?